modules/planning/proto/open_space_fallback_decider_config.proto

```diff
@@ -2,4 +2,11 @@ syntax = "proto2";
 
 package apollo.planning;
 
-message OpenSpaceFallBackDeciderConfig {}
\ No newline at end of file
+message OpenSpaceFallBackDeciderConfig {
+  // prediction time horizon for prediction obstacles
+  optional double open_space_prediction_time_horizon = 1 [default = 1.0];
```

Comment:
This is time (double in seconds) not time horizon (integer No.), please change the name to avoid confusion. Also, prediction time of default 1.0s might be too small.

Reply:
> This is time (double in seconds) not time horizon (integer No.), please change the name to avoid confusion. Also, prediction time of default 1.0s might be too small.

fixed